logstash-core/lib/logstash/outputs/base.rb

```diff
@@ -92,6 +92,10 @@ def multi_receive(events)
     end
   end
 
+  def workers_not_supported(message=nil)
+    raise "This plugin (#{self.class.name}) is using the obsolete '#workers_not_supported' method. This plugin instead should invoke 'concurrency :single' at the top of its class definition. See https://www.elastic.co/guide/en/logstash/current/_how_to_write_a_logstash_output_plugin.html for more info!"
```

Comment:
Can we amend this to say: "This plugin (#{self.class.name}) is using the obsolete '#workers_not_supported' method. If you installed this plugin specifically on this Logstash version, it is not compatible. If you are a plugin author, please see https://www.elastic.co/guide/en/logstash/current/_how_to_write_a_logstash_output_plugin.html for more info!"